migrations/versions/e3fb6631846e_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
This looks like an extra, unnecessary migration. Having this around could confuse flask, since both this and the other migration look like "first" migrations (neither has a non-None `down_revision` value, which is how flask traces back the migration dependencies). So flask could theoretically pick either as the first, and you could end up with weird situations where a deployment won't properly generate the tables on the remote server, even if it's working locally.

Try to keep an eye out for stray migrations and keep them tidy.